### PR TITLE
fixed distro_tools for other interface type.

### PR DESCRIPTION
**Pull request by smruti77**
fixed distro_tools for different interfaces.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

File: io/pci/distro_tools.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def setUp(self):
         self.option = self.params.get("test_opt", default='')
         self.tool = self.params.get("tool", default='')
         self.pci_device = self.params.get("pci_device", default='')
+        self.adapter_type = self.params.get("adapter_type", default='')
```
Comment (narasimhan-v) [inline io/pci/distro_tools.py:43]: Why do we need this ?
Can't this be found via `pci.get_pci_class_name()` https://github.com/avocado-framework/avocado/blob/781adbaa7a0736f10114fb8f5bd107f08f495eb5/avocado/utils/pci.py#L138

File: io/pci/distro_tools.py.data/distro_tools.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 pci_device: ""
+adapter_type: "net"
```
Comment (narasimhan-v) [inline io/pci/distro_tools.py.data/distro_tools.yaml:2]: Why only net here ?

Comment (smruti77) [reply io/pci/distro_tools.py.data/distro_tools.yaml:2]: just for default i gave.

File: io/pci/distro_tools.py (hunk 1 above)
Comment (smruti77) [reply io/pci/distro_tools.py:43]: ok i will use this.

File: io/pci/distro_tools.py.data/distro_tools.yaml (hunk 2 above)
Comment (narasimhan-v) [reply io/pci/distro_tools.py.data/distro_tools.yaml:2]: This is not needed if you use get_class_name()

**Pull request merged**